### PR TITLE
SD-977 - Administer build permission on pipeline security

### PR DESCRIPTION
**Pull request by rdorta-simeon**

### CODE REVIEW

File: SimeonInstaller.ps1 [hunk 1]
```diff
@@ -227,22 +227,16 @@ CRLFOption=CRLFAlways
...
-        elseif ($IsWindows) {
-            $modules = Get-Module 'AzureAD', 'AzureAD.Preview' -ListAvailable -EA SilentlyContinue
-            if (!$modules) {
-                Install-Module AzureAD -Scope CurrentUser -Force -AllowClobber -AcceptLicense -SkipPublisherCheck | Out-Null
-            }
-            else {
-                $modules | Import-Module
-            }
+
+        $modules = Get-Module 'AzureAD', 'AzureAD.Preview' -ListAvailable -EA SilentlyContinue
```
Comment (jeff-simeon) [inline SimeonInstaller.ps1:231]: why this change? AzureAD doesn't work on non-Windows OS

File: SimeonInstaller.ps1 [hunk 2]
```diff
@@ -669,10 +665,11 @@ CRLFOption=CRLFAlways
         Write-Information "Allowing $SubjectGroupPrincipalName to $PermissionDescription"
         $groupDescriptor = ($Groups |? principalName -eq $SubjectGroupPrincipalName).descriptor
         $identityDescriptor = ((Invoke-WithRetry { Invoke-RestMethod -Header $authenicationHeader -Uri "https://vssps.dev.azure.com/$Organization/_apis/identities?api-version=6.0&subjectDescriptors=$groupDescriptor" -Method Get }).value).descriptor
-        # 2e9eb7ed-3c0a-47d4-87c1-0ffdd275fd87 is the Git Repositories namespace
-        Invoke-WithRetry { Invoke-RestMethod -Header $authenicationHeader -Uri "https://dev.azure.com/$Organization/_apis/AccessControlEntries/2e9eb7ed-3c0a-47d4-87c1-0ffdd275fd87?api-version=5.0" -Method Post -ContentType "application/json" -Body @"
+        $uri = "https://dev.azure.com/$Organization/_apis/AccessControlEntries/$($ResourceId)?api-version=5.0"
+        Write-Information $uri
```
Comment (jeff-simeon) [inline SimeonInstaller.ps1:669]: why Write-Inforamtion here?

**Review by jeff-simeon — APPROVED** [review]

**Review by david-simeon — APPROVED** [review]
